irispy/tests/test_iris_tools.py

```diff
@@ -1,2 +1,99 @@
 # -*- coding: utf-8 -*-
-# """Tests for functions in iris_tools.py"""
\ No newline at end of file
+# """Tests for functions in iris_tools.py"""
```

Comment:
Delete # from start of this line.  That was my fault for putting it in the first place.